components/deploy/PreviewButton.tsx

```
@@ -0,0 +1,28 @@
+  className?: string;
+  label?: string;
+}
+
+const PreviewButton: React.FC<PreviewButtonProps> = ({ className = '', label }) => {
+  const [showPreview, setShowPreview] = useState(false);
+  return (
+    <>
+      <button
+        className={`mt-2 px-4 py-1 bg-blue-600 text-white rounded hover:bg-blue-700 transition text-sm ${className}`}
+        onClick={() => setShowPreview((v) => !v)}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)